src/styles/styles.css

```
@@ -0,0 +1,137 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.wrapper {
+  margin-right: auto;
+  margin-left: auto;
+  width: 60%;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: solid 1px #e8d2d2;
+  height: 11rem;
+}
+
+.header__title {
+  color: #827b7b;
+  text-transform: uppercase;
+  font-weight: 100;
+  font-size: 1.7rem;
+  letter-spacing: 1px;
+  margin: 0 2rem 0;
+  text-align: center;
+}
+
+form {
```

O que acha de utilizar uma classe com nome `filters`?

```suggestion
.filters {
```